rust/core-lib/src/event_context.rs

```diff
@@ -368,14 +368,8 @@ impl<'a> EventContext<'a> {
     pub(crate) fn do_plugin_update(&mut self, update: Result<Value, RpcError>,
                                     undo_group: usize) {
 
-        match update.map(serde_json::from_value::<UpdateResponse>) {
-            Ok(Ok(UpdateResponse::Edit(edit))) => {
-                let author = edit.author.clone();
-                self.editor.borrow_mut().apply_plugin_edit(edit, Some(undo_group));
-                self.after_edit(&author);
-                self.render_if_needed();
-            }
-            Ok(Ok(UpdateResponse::Ack(_))) => (),
+        match update.map(serde_json::from_value::<u64>) {
```

Comment:
here we don't actually need to parse the value at all; as long as this is _any_ `Ok(_)` we should consider the message ack'd.